src/Consolonia.Designer/Consolonia.Designer.csproj

```diff
@@ -8,6 +8,8 @@
         <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Desktop" Version="$(AvaloniaVersion)" />
         <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Fonts.Inter" Version="$(AvaloniaVersion)" />
         <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+        <PackageReference Include="Consolonia.Core" Version="11.2" />
```

Comment:
Why are we referencing as a package if already referenced as a project?

Reply:
This is the way you used to do project and package references. Now nuget always treats project references as package references, I have removed this in the packages PR, but I Will do it here too.